tlcc/README.md

```diff
@@ -7,37 +7,22 @@ https://creativecommons.org/licenses/by/4.0/
 Before your continue here make sure you have built ``tlcc_enclave``.
 We refer to [tlcc_enclave/README.md](../tlcc_enclave).
 
-First we build tlcc as system chaincode plugin. What is that?
-See [here](https://hyperledger-fabric.readthedocs.io/en/release-1.4/systemchaincode.html).
-
-    $ make
-
-This builds `tlcc.so`. Using this plugin when running the peer inside
-Docker most probably will not work out-of-the-box, thus, not supported
-right now.
-
 ## Integrate with Fabric
 
-Add `tlcc` as system chaincode plugin to your `core.yaml`. Example:
-
-```
-chaincode:
-    system:
-        tlcc: enable
+TLCC is integrated into a fabric peer as system chaincode and thus must be build together with the peer binary.
+We provide a simple Fabric for the integration that can be used as follows:
 
-    systemPlugins:
-        - enabled: true
-        name: tlcc
-        path: /path-to/fabric-private-chaincode/tlcc/tlcc.so
-        invokableExternal: true
-        invokableCC2CC: true
-```
+    $ cd $FABRIC_PATH
+    $ git am $FPC_PATH/fabric/0001-Integrate-tlcc-syscc.patch
```

Comment:
Hmm, don't we do elsehwere already a git am $FPC_PATH/fabric/*, so this should fail here?

Reply:
right - I just wanted to document that tlcc integration relies on that patch! Let me try to make this more clear

Reply:
Maybe seems to need some resolution. I supposed the "can be used" wording above implies it is not necessary to do. 

However, people will see terminal commands and blindly copy them. It does help that this is made difficult by having the leading `$`.

Reply:
I revised all readme files